### ISSUE TITLE
Pre-Requisites

### ISSUE THREAD

**Issue by Alexdc-12** [adamdriscoll/selenium-powershell#36]
Hi
Are there any other pre-requisites for this to run?
I have this working fine on a Windows 10 PC. Have tried on another windows 10 PC and it doesnt work. 
PScore 6.0 installed
Selenium installed via Install-Module Selenium
Chromedriver 76 downloaded to match Chrome install (76.0.3809.126)

SImple Script
$Driver = Start-SeChrome
Enter-SeUrl https://www.google.com -Driver $Driver

Chromedriver starts, Chrome starts with Data;, as the URL.

If you go back to ChromeDriver it says Timed Out connecting to Chrome, Retrying...

In powershell ISE i also get an error stating
Method Invocation failed because [System.String] does not contain a method named 'Navigate'.
At C:\Program Files\windowsPowershell\Modules\Selenium\1.2\Selenium.psm1:65 char :5
+         $Driver.Navigate().GoToUrl($url)

**Comment by BenjaminBaxleyINAP** [comment]
Hi @Alexdc-12 ,
I'm taking a shot in the dark, that could be a blocked port. When you start, the chromedriver console window should tell you what port it's starting on (the last time I ran, it was 1370). There's also an update to the module now, it may be worth trying the updated version as well?

I hope that helps!

**Comment by the-mentor** [comment]
@Alexdc-12 ChromeDriver comes with the module.
You need to make sure you have the browser that you're going to use with the module installed.

-DM

**Comment by the-mentor** [comment]
Hi @Alexdc-12 any update on this issue?

-DM

**Comment by Alexdc-12** [comment]
Hi
I did have the same Chrome version installed as the driver, i will try this again tonight and revert back to you. 
Thanks

**Comment by Alexdc-12** [comment]
Still not working, here are steps i followed
Updated selenium by doing Install-module Selenium -force
Download chromedriver 78 to match Chrome install, placed in the assemblies folder. 

Test script above, same problem. its listening on port 55761 - I tried disabling the firewall and the issue is the same.

**Comment by the-mentor** [comment]
@Alexdc-12 can you try and clone the latest git repo and run the module from there since I've updated the ChromeDriver yesterday.
Also what OS are you running? If you are on windows are you using a 32bit install?

-DM

**Comment by Alexdc-12** [comment]
@the-mentor yes i can try that this evening, 
I am on Windows 10 64Bit.
Its worth noting the Windows 10 PC it works on is also 64Bit though.

**Comment by Alexdc-12** [comment]
Still doesnt work, i have replaced chrome with Chrome 32bit as the laptop it works on is Chrome 32bit. 
As you can see driver starts and chome starts but nothing else happens

![1](https://user-images.githubusercontent.com/48382348/69172349-9289ef80-0af5-11ea-9478-a3de08ccaac4.PNG)

if i look in Powershell ISE i see this error.
![2](https://user-images.githubusercontent.com/48382348/69172350-93228600-0af5-11ea-812a-62e7d90ec28d.PNG)

**Comment by the-mentor** [comment]
@Alexdc-12 based on the screenshots you are running the 2.2.0 version of the module not the 2.3.0 version that is in the GitHub repo.

**Comment by Alexdc-12** [comment]
@the-mentor yes, when I copied the GitHub repo and pasted over the 2.2.0 folder the script.would no longer run citing that start-sechrome is not a known command. I replaced the chrome driver from chrome driver site and web driver DLLs from the repo to do that test

**Comment by the-mentor** [comment]
@Alexdc-12 make sure you right click >properties >unblock the zip you download from GitHub since exe files inside the zip will be blocked by windows.

**Comment by Alexdc-12** [comment]
Still the same? 
![3](https://user-images.githubusercontent.com/48382348/69175158-2611ef00-0afb-11ea-9ae9-3f81bd114a22.PNG)

**Comment by Alexdc-12** [comment]
if i rename folder to 2.3.0 it allows script to run but same error as before. 
![4](https://user-images.githubusercontent.com/48382348/69175517-d2ec6c00-0afb-11ea-9333-2e87df67df7e.PNG)

**Comment by the-mentor** [comment]
@Alexdc-12 does this issue happens with firefox?
can you try to install the latest Visual C++ redistributables and try again ?
https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

-DM

**Comment by Alexdc-12** [comment]
Not sure about firefox as i dont use it. If i use the webdriver dlls instead of PS module it seems OK. 

`$YourURL = "https://www.google.co.uk" # Website we'll log t
# Invoke Selenium into our script!​
$env:PATH += ";C:\Program Files\WindowsPowerShell\Modules\Selenium\2.3.0\assemblies\" # Adds the path for ChromeDriver.exe to the environmental variable ​
Add-Type -Path "C:\Program Files\WindowsPowerShell\Modules\Selenium\1.2\assemblies\WebDriver.dll" # Adding Selenium's .NET assembly (dll) to access it's classes in this PowerShell session​
#View mode enable line below
#$ChromeDriver = New-Object OpenQA.Selenium.Chrome.ChromeDriver # Creates an instance of this class to control Selenium and stores it in an easy to handle variable​
#Headless mode, 3 lines below

while($true){

$ChromeOptions = New-Object OpenQA.Selenium.Chrome.ChromeOptions
$ChromeOptions.addArguments('headless')
$ChromeDriver = New-Object OpenQA.Selenium.Chrome.ChromeDriver($ChromeOptions)
$ChromeDriver.Navigate().GoToURL($YourURL) # Browse to the specified website`

I updated the Visual C++ redist but still the same.

**Comment by the-mentor** [comment]
@Alexdc-12 can you send me the URL in a Direct message so I can do some testing on it?

Thank you. 
-DM

**Comment by Alexdc-12** [comment]
Any URL seems to give the issue.

**Comment by itfranck** [comment]
This issue is inactive for quite a while. 
A few versions were released since then.

I will now close this issue.
If you still have the same problem with the latest version, you can open another ticket and I will look at it.
